docs/configuring-playbook-ma1sd.md

```diff
@@ -129,13 +116,27 @@ matrix_ma1sd_configuration_extension_yaml: |
             whitelist: ~
 ```
 
+## Adjusting DNS records
+
+To make the ma1sd Identity Server enable its federation features, set up an SRV record that looks like this:
+- Name: `_matrix-identity._tcp` (use this text as-is)
+- Content: `10 0 443 matrix.example.com` (replace `example.com` with your own)
+
+See [ma1sd's documentation](https://github.com/ma1uta/ma1sd/wiki/mxisd-and-your-privacy#choices-are-never-easy) for information on the privacy implications of setting up this SRV record.
+
+**Note**: This `_matrix-identity._tcp` SRV record for the identity server is different from the `_matrix._tcp` that can be used for Synapse delegation. See [howto-server-delegation.md](howto-server-delegation.md) for more information about delegation.
+
```

Comment:
Though it might be hard to see it due to its length, the reason why the DNS section was moved all down there is because the paragraphs until that belong to the `Adjusting the playbook configuration` section:

````
## Adjusting the playbook configuration
### …
### Additional features
#### …
#### …
## Adjusting DNS records
## Installing
````

On this PR, in order to keep the consistency over the whole documentation, I think the current status (as my proposal) should be fine.

As adjusting DNS records does not constitute playbook configuration, this can be changed as below by moving it over that:

````
## Adjusting DNS records
## Adjusting the playbook configuration
### …
### Additional features
#### …
#### …
## Installing
````

This can be more reasonable as 1) changing DNS records belongs to rather prerequisites, 2) it sometimes takes some time for those records to be propagated, and also 3) generally the instruction for DNS settings is fairly short and can be overlooked.

What would you think?

Reply:
Ah, right! Makes sense!

In that case, moving "Adjusting DNS records" to be above "Adjusting the playbook configuration" makes more sense. Let's do that!

It also follows the general structure of the "quick start guide" ("DNS first", ..., adjusting configuration later, .. installation).